src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -197,19 +200,20 @@ public static IMethodSymbol GetPropertyMethod (IPropertyReferenceOperation opera
 
 		public override TValue VisitPropertyReference (IPropertyReferenceOperation operation, LocalState<TValue> state)
 		{
-			if (operation.Instance != null) {
-				var instanceValue = Visit (operation.Instance, state);
-				HandleReceiverArgument (instanceValue, GetPropertyMethod (operation), operation);
+			var propertyMethod = GetPropertyMethod (operation);
```

Comment:
I know you're just copying this over but fyi there's an IOpertionExtensions.GetValueUsageInfo that provides all this information and much more

Reply:
I changed it, could you please review this bit if it makes sense?